WTWSMS/history/characters/aramean.txt

```diff
@@ -24,12 +24,14 @@
 	}
 }
 154000 = {
-	name="Le�ntios"
+	name="Leontios"
```

Comment:
WAD spelling, not a typo

Reply:
Reverted. Are you sure this displays properly in game?

Reply:
Not with your change.

Reply:
Could you revert it yourself? I copied from the master branch and somehow it's still different.

Reply:
Yes, but copied via GitHub? That never works, just for the information.

Reply:
No, via Notepad++